app/common/FirebaseService.ts

```
@@ -25,110 +24,159 @@
+  private async initialize(): Promise<void> {
+    await Promise.all([this.fetchBytes(), this.fetchNibbles()]);
+    this.isInitialized = true;
+  }
 
-    const queryResults: DocumentData[] = await getDocs(q).then(
-      (response: QuerySnapshot<DocumentData, DocumentData>) =>
-        response.docs.map(
-          (doc: QueryDocumentSnapshot<DocumentData, DocumentData>) => doc.data()
-        )
-    );
+  private async fetchBytes(): Promise<void[]> {
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)